pyaedt/application/Analysis.py

```diff
@@ -408,6 +408,174 @@ def SolutionTypes(self):
         """
         return SOLUTIONS()
 
+    @aedt_exception_handler
+    def analyze_all(self):
+        """Analyze all setup in a actual design.
+
+        Returns
+        -------
+        bool
+            ``True`` when simulation is finished.
+        """
+        self.odesign.AnalyzeAll()
+        return True
+
+    @aedt_exception_handler
+    def list_of_variations(self, setupname=None, sweepname=None):
```

Comment:
```suggestion
    def list_of_variations(self, setup_name=None, sweep_name=None):
```

It is just to respect the snake_case convention.